system/Log/Handlers/ErrorlogHandler.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * This file is part of the CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Log\Handlers;
+
+/**
+ * Log error messages to PHP error log
+ */
+class ErrorlogHandler extends BaseHandler
+{
+
+	/**
+	 * Constructor
+	 *
+	 * @param array $config
+	 */
+	public function __construct(array $config = [])
+	{
+		parent::__construct($config);
+	}
+
+	//--------------------------------------------------------------------
+
```

Comment:
```suggestion
```

No need to call parent constructor if no special initializations will be made by extending class.